### ISSUE TITLE
Apple Intelligence is not intelligent (AKA it's stuck on downloading)

### ISSUE THREAD

**Issue by BilalFunker2704** [leminlimez/Nugget#152]
So i'm working with an IPhone 11 and tried using Apple Intelligence, after spoofing and enabling Face ID didn't work, which is normal enough.

 so i disregarded that and installed Apple Intelligence, as it was installing it reached 100%, then the % went away but the downloading text stayed.

 i waited an hour, then i got tired and un-spoofed it, it was still stuck on downloading AFTER un-spoofing.

even redoing the process after removing all tweaks got the same results, the ONLY way i could remove this was to restore from a backup i made before using nugget, and i tried all of this about 3 times and yielded the same results, could i get any help please.

or at least just fix the bug, i'm on an IPhone 11 using IOS 18.2 beta (2?).

**Comment by loqimean** [comment]
you need to remove all tweaks ONLY from the special button in the app!
<img width="1002" alt="Screenshot 2024-11-11 at 18 16 04" src="https://github.com/user-attachments/assets/b8a52b57-dcb6-42b9-a634-4ddedb6c2560">

**Comment by BilalFunker2704** [comment]
> you need to remove all tweaks ONLY from the special button in the app! <img alt="Screenshot 2024-11-11 at 18 16 04" width="1002" src="https://private-user-images.githubusercontent.com/49816584/384991833-b8a52b57-dcb6-42b9-a634-4ddedb6c2560.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEzNTYyODUsIm5iZiI6MTczMTM1NTk4NSwicGF0aCI6Ii80OTgxNjU4NC8zODQ5OTE4MzMtYjhhNTJiNTctZGNiNi00MmI5LWE2MzQtNGRkZWRiNmMyNTYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTExVDIwMTMwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1YjliNjkyNzNkMzBjYWRjYTBiOGM2NTViMjE4ZDQ2YjI0ZTZhZmIyOGY1ZDhjMmQ4ZTA2NGEzNGVjOGViNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.4ktdUtecdaXz0rlJw6HzPEH3HLjbEiPvX3JW3Y7dags">

Is this a fix for the Face ID or for having to restore from backup?

**Comment by loqimean** [comment]
> > you need to remove all tweaks ONLY from the special button in the app! <img alt="Screenshot 2024-11-11 at 18 16 04" width="1002" src="https://private-user-images.githubusercontent.com/49816584/384991833-b8a52b57-dcb6-42b9-a634-4ddedb6c2560.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEzNTYyODUsIm5iZiI6MTczMTM1NTk4NSwicGF0aCI6Ii80OTgxNjU4NC8zODQ5OTE4MzMtYjhhNTJiNTctZGNiNi00MmI5LWE2MzQtNGRkZWRiNmMyNTYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTExVDIwMTMwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1YjliNjkyNzNkMzBjYWRjYTBiOGM2NTViMjE4ZDQ2YjI0ZTZhZmIyOGY1ZDhjMmQ4ZTA2NGEzNGVjOGViNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.4ktdUtecdaXz0rlJw6HzPEH3HLjbEiPvX3JW3Y7dags">
> 
> 
> 
> Is this a fix for the Face ID and AI being stuck or for having to restore from backup?

Yes,that's fix for Face ID, but AI stuck, so, you have bricked device cause restored device instead of just restoring device params from nugget and restoring mobile gestalt
